api/operator.go

```diff
@@ -43,6 +43,26 @@ type RaftConfiguration struct {
 	Index uint64
 }
 
+// keyringRequest is used for performing Keyring operations
+type keyringRequest struct {
+	Key string `json:",omitempty"`
```

Comment:
I can't remember why I added this - it's the only field in the struct and we only ever decode to it, so there's no reason for it to be in there